pydtk/models/rosbag2.py

```diff
@@ -329,20 +329,27 @@ def generate_contents_meta(cls, path, content_key="content"):
             dict: Contents metadata
 
         """
```

Comment:
Please rollback this change to align the format of `contents` with other models.

Reply:
fixed